includes/renderers/class-sensei-renderer-single-post.php

```diff
@@ -120,13 +120,16 @@ public function render() {
 	 * Create the posts query.
 	 */
 	private function setup_post_query() {
+		global $page;
```

Comment:
`WP_Query` was not taking into account the page number so we were always displaying the first part of the content when calling `the_content` in the templates.

@alexsanford do you remember any reason why this was being done like that?

Reply:
Cool! This should be ready to go then.

Reply:
Let's go! 🚀